<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MvpAdvancedWidget</class>
 <widget class="QWidget" name="MvpAdvancedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MvpAdvancedWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gb_address">
     <property name="title">
      <string>Address</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="1" column="6">
       <widget class="QSpinBox" name="spin_a1_begin">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Area</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>End</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="8" rowspan="2">
       <widget class="QLabel" name="label_bytes">
        <property name="text">
         <string>0</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Bytes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7" rowspan="2">
       <widget class="QSpinBox" name="spin_len">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSpinBox" name="spin_a1_end">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Section</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>A2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spin_a2_end">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>A1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spin_a2_begin">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Pages</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_area">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_section"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_dec">
        <property name="text">
         <string>Dec</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_hex">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="4">
      <widget class="QPushButton" name="pb_keys_info">
       <property name="text">
        <string>Device Info</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_boot">
       <property name="toolTip">
        <string>Boot selected area index</string>
       </property>
       <property name="statusTip">
        <string>Boot selected area index</string>
       </property>
       <property name="text">
        <string>Boot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_load_from_file">
       <property name="toolTip">
        <string>Write file contents to selected start address</string>
       </property>
       <property name="statusTip">
        <string>Write file contents to selected start address</string>
       </property>
       <property name="text">
        <string>Load from file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_save_to_file">
       <property name="toolTip">
        <string>Save selected memory range to file</string>
       </property>
       <property name="statusTip">
        <string>Save selected memory range to file</string>
       </property>
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_nop_recovery">
       <property name="text">
        <string>NOP recovery</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_dump_to_console">
       <property name="toolTip">
        <string>Dump selected memory range to the console</string>
       </property>
       <property name="statusTip">
        <string>Dump selected memory range to the console</string>
       </property>
       <property name="text">
        <string>Dump to console</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pb_rdi">
       <property name="text">
        <string>Hardware ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_erase_section">
       <property name="text">
        <string>Erase Section</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pb_mdpp16_cal">
       <property name="text">
        <string>MDPP16 cal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pb_mdpp32_cal">
       <property name="text">
        <string>MDPP32 cal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pb_manage_keys">
       <property name="text">
        <string>Manage Keys</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pb_dip_switches">
       <property name="text">
        <string>DIP Switches</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
